apps/nowcasting-app/components/charts/forecast-header/ui.tsx

```diff
@@ -75,12 +94,18 @@ const ForecastHeaderUI: React.FC<ForecastHeaderProps> = ({
           <ForecastWithActualPV
             forecast={`${forcastPV}`}
             pv={`${actualPV}`}
+            tip={`Nowcast / Actual PV`}
```

Comment:
As per your comment/description on this PR, not 100% myself but I think I would probably go with "Forecast" here and below, just for consistency with other labels on the page?

Reply:
thanks. that makes sense.  I updated the labels to reflect the naming conventions on the page.

Reply:
<img width="302" alt="Screenshot 2022-10-14 at 18 13 28" src="https://user-images.githubusercontent.com/86949265/195893000-be245e5d-3451-443e-a685-e947012329cb.png">

<img width="367" alt="Screenshot 2022-10-14 at 18 16 18" src="https://user-images.githubusercontent.com/86949265/195893477-f20b4442-c4c5-42fc-b7a9-ad7488277e1b.png">